tgui/packages/tgui/interfaces/Terminal.js

```diff
@@ -0,0 +1,80 @@
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { Box, Input, Button, Section } from 'tgui/components';
+import { ColorBox, Flex } from '../components';
+
+export const Terminal = (props, context) => {
+  const { act, data } = useBackend(context);
+  const peripherals = data.peripherals || [];
+  const { textInput } = props;
+
+  const {
+    fdisk,
+    idcard,
+    displayHTML,
+    TermActive,
+    windowName,
+    user,
+    fontColor,
+    bgColor,
+    diskdrive,
+  } = data;
+
+  return (
+    <Window
+      theme="retro-dark"
+      title={windowName}
+      width="380"
+      fontFamily="Consolas"
+      font-size="10pt"
+      height="350">
+      <Window.Content>
+        <Section backgroundColor="#0f0f0f" scrollable fill height="70%" >
```

Comment:
Need to dive in deeper to validate, but I suspect you probably want to use `Stack` (and `Stack.Item`) around the various `Section` components to do layout better, rather than this weird height and margin setting?